src/lib/auth.ts

```diff
@@ -38,6 +38,7 @@ export const authOptions: NextAuthOptions = {
 
       token.id = foundUser.id
       token.profileId = foundUser.profile?.id
+      token.profile = !!foundUser.profile
```

Comment:
**issue:** But we had to remove `profileId` from session because it leads to critical bug. Adding `profile` to session is basically the same problem